### PR TITLE
Experiment with cicero

### PR DESCRIPTION
**Pull request by Anviking**
### Issue Number

ADP-2438

### CODE REVIEW

File: lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs [hunk 1]
```diff
@@ -2346,6 +2346,9 @@ balanceTransactionSpec = describe "balanceTransaction" $ do
 
     balanceTransactionGoldenSpec
 
+    it "<test failure>" $ do
+        True `shouldBe` False
```
Comment (jonathanknowles) [inline lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs]: 😄

**Comment by Anviking** [comment]
Compilation error [2653500](https://github.com/input-output-hk/cardano-wallet/pull/3601/commits/26535002c9401e6c54f0d5d0819756839036424c) appears in logs of `ci/pr (linux)`

```
2022-12-01 11:42:49 | 2022-12-01T11:42:47.059211201+00:00 stderr F cardano-wallet-lib-cardano-wallet> � Variable not in scope: cp :: Wallet s
2022-12-01 11:42:49 | 2022-12-01T11:42:47.059330133+00:00 stderr F cardano-wallet-lib-cardano-wallet> � Perhaps you meant �cp0� (line 840)
2022-12-01 11:42:49 | 2022-12-01T11:42:47.059330133+00:00 stderr F cardano-wallet-lib-cardano-wallet> \|
2022-12-01 11:42:49 | 2022-12-01T11:42:47.059330133+00:00 stderr F cardano-wallet-lib-cardano-wallet> 843 \| pure (cp, meta, Set.fromList (fromTransactionInfo <$> pending))
2022-12-01 11:42:49 | 2022-12-01T11:42:47.087979211+00:00 stderr F cardano-wallet-lib-cardano-wallet> \| ^^
```
It is also failing with status code 1. This seems promising. It was failing with status code 137 before, and at one point succeeding (in #3594 )

**Comment by Anviking** [comment]
bors try

File: lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs (hunk 1 above)
Comment (HeinrichApfelmus) [reply lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs]: In this case, ignorance really is strength — you don't want to know. 🤓

**Comment by disassembler** [comment]
`cardano-node: pokeSockAddr: path is too long`

The integration test appears to be launching cardano-node with too long of a socket path.

**Comment by Anviking** [comment]
bors try

**Pull request closed without merging**